projenrc/support.ts

```diff
@@ -2,13 +2,14 @@ import { JsonFile, Project } from 'projen';
 import type { ReleasesDocument } from '../src/support';
 
 export const SUPPORT_POLICY: ReleasesDocument = {
-  current: '5.4',
+  current: '5.5',
   maintenance: {
     // version: End-of-support date
     '5.0': new Date('2024-01-31'),
     '5.1': new Date('2024-02-28'),
     '5.2': new Date('2024-06-30'),
     '5.3': new Date('2024-10-15'),
+    '5.4': new Date('2024-02-19'),
   },
```

Comment:
We tend to round up to the of the month. Partly to be nice to users, partly because you don't know when this PR gets merged and then you don't have to keep updating the date. 

We should probably also include this in the instructions. 😅